src/openrct2/drawing/TTFSDLPort.cpp

```diff
@@ -995,6 +995,7 @@ void TTF_CloseFont(TTF_Font* font)
             fclose(font->src);
         }
         free(font);
+        delete font;
```

Comment:
You should either delete or free the font. Never both. Also, you gotta make sure whoever created this font used `new` for `delete` to be used.